src/guide/jbs-jdk-bug-system.md

```diff
@@ -431,10 +431,10 @@ This table contains some frequently used JBS labels and their meaning. Please he
                            [*(Rel)*[-fix-yes]{.jbs-label}]{#rel-fix-yes}<br />
                            [*(Rel)*[-fix-no]{.jbs-label}]{#rel-fix-no}</td>
     <td class="dictionary">
-      Used in ramp down phase 2 to indicate that an issue would be of interest to get integrated into release *(Rel)*. E.g., [jdk12u-fix-request]{.jbs-label}
+      *(Rel)*[-fix-request]{.jbs-label} is used in ramp down phase 2 to indicate that an issue would be of interest to be integrated into release *(Rel)*. E.g., [jdk12-fix-request]{.jbs-label}
```

Comment:
Should the section be expanded for update releases? The labels like `jdk12u-fix-request` and `jdk24u-fix-request` are constantly used to [request approval for backporting](https://openjdk.org/guide/#requesting-approvals-for-backports) an update release.

The `jdk##-fix-request` and `jdk##u-fix-request` are similar, but they have a different role in the OpenJDK Project.

Reply:
I'd be happy to include such an update. Could you write the text, either here in a comment or as a separate change?

Reply:
Yeah, it makes sense to expand the section in another PR.